Godeps/Godeps.json

```diff
@@ -0,0 +1,5 @@
+{
+	"ImportPath": "github.com/mroth/subtleist",
+	"GoVersion": "go1.5.1",
```

Comment:
I *strongly* recommend using Go 1.5's `-vendor` flag or the largely similar [gb](https://github.com/constabulary/gb) over Godeps, but your call.